docs/user/api/bigquery.md

```diff
@@ -0,0 +1,32 @@
+# BigQuery Datasets
+
+We use BigQuery to serve our public datasets. PyPI offers two tables whose
```

Comment:
Thinking out loud: it might be nice/useful to link to @sethmlarson's [pypi-data](https://github.com/sethmlarson/pypi-data) as a local/cheaper alternative to BQ for some use cases. 

LMK if you'd like that; I've left it out so far since the current changes are mostly a straight conversion 🙂

Reply:
I like the project but I'm concerned about two things here: 1) there are other similar projects, and we'd have to justify which ones we include in the docs & don't include 2) up-to-date-edness, e.g. looks like that project is missing the new `License-File` and `License-Expression` fields.